cmd/skaffold/app/cmd/flags.go

```diff
@@ -147,6 +147,14 @@ var flagRegistry = []Flag{
 		FlagAddMethod: "StringVar",
 		DefinedOn:     []string{"all"},
 	},
+	{
+		Name:          "sync-remote-cache",
+		Usage:         "One of `always`(default), `missing`, or `never`.\nSet to `always` to sync remote git repositories cache in `remote-cache-dir` directory every time skaffold starts.\nSet to `missing` to only clone the remote git repositories once if they don't exist locally in the `remote-cache-dir` directory.\nSet to `never` to manually manage the remote repositories.",
```

Comment:
Cobra's long-line handling is unfortunately non-existent.  But the use of `\n` doesn't render well:
```
  -q, --quiet=false: Suppress the build output and print image built on success. See --output to format output.
      --remote-cache-dir='': Specify the location of the git repositories cache (default $HOME/.skaffold/repos)
      --rpc-http-port=50052: tcp port to expose event REST API over HTTP
      --rpc-port=50051: tcp port to expose event API
      --skip-tests=false: Whether to skip the tests after building
      --sync-remote-cache='always': One of `always`(default), `missing`, or `never`.
Set to `always` to sync remote git repositories cache in `remote-cache-dir` directory every time skaffold starts.
Set to `missing` to only clone the remote git repositories once if they don't exist locally in the `remote-cache-dir` directory.
Set to `never` to manually manage the remote repositories.
  -t, --tag='': The optional custom tag to use for images which overrides the current Tagger configuration
      --toot=false: Emit a terminal beep after the deploy is complete
```

I think we should go with the flow and do wrapping consistently.

```suggestion
		Usage:         "Controls how Skaffold manages the remote config cache (see `remote-cache-dir`). One of `always` (default), `missing`, or `never`. `always` syncs remote repositories to latest on access. `missing` only clones remote repositories if they do not exist locally. `never` means the user takes responsibility for updating remote repositories.",
```